src/components/UserInfo.js

```diff
@@ -2,18 +2,23 @@ export default class UserInfo {
   constructor(selectors) {
     this._nameSelector = selectors.nameSelector;
     this._jobSelector = selectors.jobSelector;
+    this._avatarSelector = selectors.avatarSelector;
     this._name = document.querySelector(this._nameSelector);
     this._job = document.querySelector(this._jobSelector);
+    this._avatar = document.querySelector(this._avatarSelector);
   }
 
   getUserInfo() {
     return {
       name: this._name.textContent,
-      job: this._job.textContent,
+      about: this._job.textContent,
     };
   }
-  setUserInfo({ name, title }) {
+  setUserInfo({ name, about, avatar, _id }) {
+    console.log(avatar);
     this._name.textContent = name;
-    this._job.textContent = title;
+    this._job.textContent = about;
+    this._avatar.src = avatar;
```

Comment:
It's needed to create a separate setter for avatar `setAvatar` and use it when needed to update the avatar, because user info and avatar are controlled by different forms.

When initializing user info you can use bit methods:

```js
userInfo.setUserInfo({ title, description })
userInfo.setAvatar(avatar)
```